setup.cfg

```diff
@@ -50,6 +50,7 @@ test =
 examples =
     cartopy==0.19.0.post1  # for WMS example
     scipy >=1.7,<2 # for WMS example
+    xgboost >= 1.7, <2 # this needs xgboost lib on the system
```

Comment:
ist it still possible to run the other examples if the xgboost lib is not installed? I mean, does `python3 -m pip install -e .[examples]` fail if the lib is not installed?

Reply:
It should not be a problem, because the python package comes with a prebuilt binary for xgboost [xgboost documentation](https://xgboost.readthedocs.io/en/stable/install.html#stable-release).
I didn't think about this when I added the dependency, but I guess the comment can be removed. As long as the c dependencies are provided (which should be covered by gdal anyway), everything should be fine.